lib/internal/Magento/Framework/Message/Manager.php

```diff
@@ -79,16 +85,18 @@ public function __construct(
         Event\ManagerInterface $eventManager,
         LoggerInterface $logger,
         $defaultGroup = self::DEFAULT_GROUP,
-        ?ExceptionMessageFactoryInterface $exceptionMessageFactory = null
+        ?ExceptionMessageFactoryInterface $exceptionMessageFactory = null,
+        ?ExceptionMessageLookupFactory $exceptionMessageLookupFactory = null
```

Comment:
@dimitriBouteille, I just wonder why it's needed.

Reply:
I don't kown, is not my code 😂

Reply:
@engcom-Charlie could you please explain this?

Reply:
@ihor-sviziev on prior implementation, in https://github.com/magento/magento2/pull/39153/commits/0c33ab925541ca1cc1ab68c1bfaf92fe97ba58ae, PR was showing [static](https://public-results-storage-prod.magento-testing-service.engineering/reports/magento/magento2/pull/39153/88de68b01ca0359a43b11b6cac8973b3/Statics/console-error-logs.html) test failures which we have conveyed in [here](https://github.com/magento/magento2/pull/39153#pullrequestreview-2506034221). We have fixed them in [here](https://github.com/magento/magento2/commit/c8abbb92ac1b581d388ce1875e5e925b4249187f)

Reply:
As desribed in https://github.com/magento/magento2/pull/39153/files#r1950728162, this change should be reverted.

```suggestion
        ?ExceptionMessageFactoryInterface $exceptionMessageFactory = null
```